networking/firewall/000infralib/src/lib/sync.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+###########################################################
+# multihost sync
```

Comment:
Since @rasibley mentioned removing multihost parts, shouldn't this file be removed too?

Reply:
Sure, file sync.sh is removed.